### ISSUE TITLE
Failed to build

### ISSUE THREAD

**Issue by cfilipov** [kasei/kineo-endpoint#1]
Following the instructions on the README

    swift build -c release

This fails due to missing openssl, `brew install libressl` resolves that but then I get a new build error:

Strange that it's complaining about os_signpost only being available on 10.14. I'm on 10.14 (18A391).

<details>
<summary>Build output</summary>

<pre>
Cristians-MacBook-Pro:kineo-endpoint cfilipov$ swift build -c release
Compile Swift Module 'HDT' (6 sources)
Compile Swift Module 'NIOWebSocket' (9 sources)
Compile Swift Module 'Logging' (4 sources)
Compile Swift Module 'Validation' (18 sources)
Compile Swift Module 'Multipart' (8 sources)
Compile Swift Module 'URLEncodedForm' (8 sources)
Compile Swift Module 'Service' (20 sources)
Compile Swift Module 'HTTP' (25 sources)
Compile Swift Module 'Crypto' (19 sources)
Compile Swift Module 'Routing' (12 sources)
Compile Swift Module 'TemplateKit' (41 sources)
Compile Swift Module 'DatabaseKit' (30 sources)
Compile Swift Module 'Console' (28 sources)
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:59:15: error: 'init(subsystem:category:)' is only available on OS X 10.14 or newer
    let log = OSLog(subsystem: "us.kasei.swift.hdt", category: .pointsOfInterest)
              ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:100:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read shared blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:100:9: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read shared blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:100:22: error: 'begin' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read shared blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:100:22: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read shared blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:105:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read shared blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:105:9: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read shared blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:105:22: error: 'end' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read shared blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:105:22: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read shared blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:109:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read subjects blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:109:9: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read subjects blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:109:22: error: 'begin' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read subjects blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:109:22: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read subjects blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:114:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read subjects blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:114:9: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read subjects blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:114:22: error: 'end' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read subjects blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:114:22: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read subjects blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:118:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read predicates blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:118:9: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read predicates blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:118:22: error: 'begin' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read predicates blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:118:22: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read predicates blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:123:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read predicates blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:123:9: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read predicates blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:123:22: error: 'end' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read predicates blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:123:22: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read predicates blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:127:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read objects blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:127:9: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read objects blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:127:22: error: 'begin' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read objects blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:127:22: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Read objects blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:132:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read objects blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:132:9: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read objects blocks")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:132:22: error: 'end' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read objects blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:132:22: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Read objects blocks")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:90:5: note: add @available attribute to enclosing initializer
    init(metadata: DictionaryMetadata, size: Int, ptr mmappedPtr: UnsafeMutableRawPointer) throws {
    ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:225:17: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
                os_signpost(.event, log: log, name: "Dictionary", "%{public}s cache reached cache maximum size of %{public}d", "\(lookup.position)", count)
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:225:17: note: add 'if #available' version check
                os_signpost(.event, log: log, name: "Dictionary", "%{public}s cache reached cache maximum size of %{public}d", "\(lookup.position)", count)
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:220:18: note: add @available attribute to enclosing instance method
    private func updateCache(for lookup: TermLookupPair, dictionary: [Int64:Term]) {
                 ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:225:30: error: 'event' is only available on OS X 10.14 or newer
                os_signpost(.event, log: log, name: "Dictionary", "%{public}s cache reached cache maximum size of %{public}d", "\(lookup.position)", count)
                             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:225:30: note: add 'if #available' version check
                os_signpost(.event, log: log, name: "Dictionary", "%{public}s cache reached cache maximum size of %{public}d", "\(lookup.position)", count)
                             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:220:18: note: add @available attribute to enclosing instance method
    private func updateCache(for lookup: TermLookupPair, dictionary: [Int64:Term]) {
                 ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:290:22: error: 'applyingTransform(_:reverse:)' is only available on OS X 10.11 or newer
        let s = orig.applyingTransform(transform, reverse: true)!
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:290:22: note: add 'if #available' version check
        let s = orig.applyingTransform(transform, reverse: true)!
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:286:18: note: add @available attribute to enclosing instance method
    private func unicodeUnescaped(_ orig: String) -> String {
                 ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/Dictionary.swift:57:20: note: add @available attribute to enclosing class
public final class HDTLazyFourPartDictionary : HDTDictionaryProtocol {
                   ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:61:15: error: 'init(subsystem:category:)' is only available on OS X 10.14 or newer
    let log = OSLog(subsystem: "us.kasei.swift.hdt", category: .pointsOfInterest)
              ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:45:14: note: add @available attribute to enclosing class
public class HDT {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:223:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Triples", "Read ID Triples")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:223:9: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Triples", "Read ID Triples")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:220:17: note: add @available attribute to enclosing instance method
    public func triples() throws -> AnyIterator<Triple> {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:197:8: note: add @available attribute to enclosing extension
public extension HDT {
       ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:223:22: error: 'begin' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Triples", "Read ID Triples")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:223:22: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Triples", "Read ID Triples")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:220:17: note: add @available attribute to enclosing instance method
    public func triples() throws -> AnyIterator<Triple> {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:197:8: note: add @available attribute to enclosing extension
public extension HDT {
       ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:227:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Triples", "Read ID Triples")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:227:9: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Triples", "Read ID Triples")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:220:17: note: add @available attribute to enclosing instance method
    public func triples() throws -> AnyIterator<Triple> {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:197:8: note: add @available attribute to enclosing extension
public extension HDT {
       ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:227:22: error: 'end' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Triples", "Read ID Triples")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:227:22: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Triples", "Read ID Triples")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:220:17: note: add @available attribute to enclosing instance method
    public func triples() throws -> AnyIterator<Triple> {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:197:8: note: add @available attribute to enclosing extension
public extension HDT {
       ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:231:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Triples", "Materializing")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:231:9: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Triples", "Materializing")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:220:17: note: add @available attribute to enclosing instance method
    public func triples() throws -> AnyIterator<Triple> {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:197:8: note: add @available attribute to enclosing extension
public extension HDT {
       ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:231:22: error: 'begin' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Triples", "Materializing")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:231:22: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Triples", "Materializing")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:220:17: note: add @available attribute to enclosing instance method
    public func triples() throws -> AnyIterator<Triple> {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:197:8: note: add @available attribute to enclosing extension
public extension HDT {
       ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:235:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Triples", "Materializing")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:235:9: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Triples", "Materializing")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:220:17: note: add @available attribute to enclosing instance method
    public func triples() throws -> AnyIterator<Triple> {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:197:8: note: add @available attribute to enclosing extension
public extension HDT {
       ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:235:22: error: 'end' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Triples", "Materializing")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:235:22: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Triples", "Materializing")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:220:17: note: add @available attribute to enclosing instance method
    public func triples() throws -> AnyIterator<Triple> {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDT.swift:197:8: note: add @available attribute to enclosing extension
public extension HDT {
       ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:12:15: error: 'init(subsystem:category:)' is only available on OS X 10.14 or newer
    let log = OSLog(subsystem: "us.kasei.swift.hdt", category: .pointsOfInterest)
              ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:58:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Header", "Begin")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:58:9: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Header", "Begin")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:58:22: error: 'begin' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Header", "Begin")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:58:22: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Header", "Begin")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:63:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Header", "Finished")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:63:9: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Header", "Finished")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:63:22: error: 'end' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Header", "Finished")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:63:22: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Header", "Finished")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:67:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Begin")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:67:9: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Begin")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:67:22: error: 'begin' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Begin")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:67:22: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Dictionary", "Begin")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:72:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Finished")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:72:9: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Finished")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:72:22: error: 'end' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Finished")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:72:22: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Dictionary", "Finished")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:76:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Triples", "Begin")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:76:9: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Triples", "Begin")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:76:22: error: 'begin' is only available on OS X 10.14 or newer
        os_signpost(.begin, log: log, name: "Parsing Triples", "Begin")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:76:22: note: add 'if #available' version check
        os_signpost(.begin, log: log, name: "Parsing Triples", "Begin")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:80:9: error: 'os_signpost(_:dso:log:name:signpostID:_:_:)' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Triples", "Finished")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:80:9: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Triples", "Finished")
        ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:80:22: error: 'end' is only available on OS X 10.14 or newer
        os_signpost(.end, log: log, name: "Parsing Triples", "Finished")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:80:22: note: add 'if #available' version check
        os_signpost(.end, log: log, name: "Parsing Triples", "Finished")
                     ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:52:17: note: add @available attribute to enclosing instance method
    public func parse() throws -> HDT {
                ^
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-hdt.git--3041970972876117066/Sources/HDT/HDTParser.swift:10:14: note: add @available attribute to enclosing class
public class HDTParser {
             ^
error: terminated(1): /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-build-tool -f /Users/cfilipov/Workspace/kineo-endpoint/.build/release.yaml main output:
    

Cristians-MacBook-Pro:kineo-endpoint cfilipov$ 
</pre>
</details>

**Comment by kasei** [comment]
Can you try building with the target manually set to 10.14?

`swift build -Xswiftc "-target" -Xswiftc "x86_64-apple-macosx10.14" -c release`

**Comment by cfilipov** [comment]
That worked (got two warnings BTW).

```
$ swift build -Xswiftc "-target" -Xswiftc "x86_64-apple-macosx10.14" -c release
Compile Swift Module 'PathKit' (1 sources)
Compile Swift Module 'NIOPriorityQueue' (2 sources)
Compile Swift Module 'NIOConcurrencyHelpers' (2 sources)
Compile Swift Module 'Debugging' (3 sources)
Compile Swift Module 'COperatingSystem' (1 sources)
Compile Swift Module 'SPARQLSyntax' (14 sources)
Compile Swift Module 'CryptoSwift' (74 sources)
Compile Swift Module 'NIO' (55 sources)
Compile Swift Module 'URITemplate' (1 sources)
Compile Swift Module 'NIOFoundationCompat' (1 sources)
Compile Swift Module 'NIOTLS' (3 sources)
Compile Swift Module 'NIOHTTP1' (9 sources)
Compile Swift Module 'Bits' (12 sources)
Compile Swift Module 'Async' (15 sources)
Compile Swift Module 'Random' (4 sources)
Compile Swift Module 'Core' (25 sources)
Compile Swift Module 'NIOOpenSSL' (15 sources)
/Users/cfilipov/Workspace/kineo-endpoint/.build/checkouts/swift-nio-ssl.git--6632932180070619516/Sources/NIOOpenSSL/OpenSSLHandler.swift:270:35: warning: 'changeCapacity(to:)' is deprecated: changeCapacity has been replaced by reserveCapacity
                    receiveBuffer.changeCapacity(to: receiveBuffer.capacity + 1024)
                                  ^
Compile Swift Module 'NIOWebSocket' (9 sources)
Compile Swift Module 'HDT' (6 sources)
Compile Swift Module 'Kineo' (34 sources)
Compile Swift Module 'Logging' (4 sources)
Compile Swift Module 'URLEncodedForm' (8 sources)
Compile Swift Module 'Service' (20 sources)
Compile Swift Module 'Validation' (18 sources)
Compile Swift Module 'HTTP' (25 sources)
Compile Swift Module 'Multipart' (8 sources)
Compile Swift Module 'Crypto' (19 sources)
Compile Swift Module 'Routing' (12 sources)
Compile Swift Module 'TemplateKit' (41 sources)
Compile Swift Module 'DatabaseKit' (30 sources)
Compile Swift Module 'Console' (28 sources)
Compile Swift Module 'WebSocket' (6 sources)
Compile Swift Module 'Command' (16 sources)
Compile Swift Module 'Vapor' (75 sources)
Compile Swift Module 'KineoEndpoint' (1 sources)
Compile Swift Module 'kineo_endpoint' (1 sources)
Compile Swift Module 'kineo_create_db' (1 sources)
Linking ./.build/x86_64-apple-macosx10.10/release/kineo-create-db
/Users/cfilipov/Workspace/kineo-endpoint/Sources/kineo-endpoint/main.swift:137:11: warning: unnecessary check for 'OSX'; enclosing scope ensures guard will always be true
} else if #available(OSX 10.12, *) {
          ^
/Users/cfilipov/Workspace/kineo-endpoint/Sources/kineo-endpoint/main.swift:135:1: note: enclosing scope here
if #available(OSX 10.14, *) {
^
Linking ./.build/x86_64-apple-macosx10.10/release/kineo-endpoint
```

**Comment by kasei** [comment]
OK. That's good to see. I'll try to make some adjustments so that the build instructions allow building cleanly/simply in more diverse environments. Hopefully explicitly setting the build target as above has unblocked you for now.

Also, just a note, the file-storage implementation in Kineo isn't production ready. I'm confident in the memory-storage, but depending on what you're doing with Kineo, just be aware that there are some rough edges and corner cases that aren't well developed yet.

**Comment by cfilipov** [comment]
At this point I'm just playing around. I saw your note in another issue warning about production readiness. This project and the related ones are amazing, I can't believe they even exist. For the most part semantic web is a niche that doesn't seem to get a lot of attention so I was surprised as hell that I was able to find not just a wrapper but a full blown rdf and sparql implementation in Swift! Not only that but I'm actually able to follow along most of the cleanly written code. Thank you for making this open source!

**Comment by kasei** [comment]
OK. Happy to have people finding and appreciating my code! I'm always interested to know how it's being used. If you have any other issues, bug reports, or feature requests, please don't hesitate to get in touch!

**Comment by cfilipov** [comment]
As for how it's being used (or what I'm investigating): I've been looking for an easily embeddable triplestore to use in place of SQLite. It looks like spaqrl gives me everything I love about sql with way more flexibility. I am using a small subset of the [FMA dataset](https://en.wikipedia.org/wiki/Foundational_Model_of_Anatomy) to generate detailed workout metrics. I made it work in SQL but it was rather painful. Even for use cases that don't involve graphs I still think rdf and sparql could be beneficial.
